stacks/apps/augury.yml

```diff
@@ -369,6 +399,12 @@ Resources:
               Value: !Ref SecretsVersion
             - Name: AWS_DEFAULT_REGION
               Value: !Ref AWS::Region
+            - Name: UPLOAD_ADFILE_SECRET_ACCESS_KEY
```

Comment:
Hrm,  now that I think about it, I wonder if these will be patched into the container.  I suppose the container is launched with the addition of these ECS env variables?

Reply:
Yes, those are passed in to ` --env` of `docker run`